tests/src/App/IndexedName.cpp

```diff
@@ -301,6 +301,16 @@ TEST_F(IndexedNameTest, comparisonOperators)
     EXPECT_FALSE(indexedName9 == indexedName10);
     EXPECT_TRUE(indexedName9 != indexedName10);
     EXPECT_FALSE(indexedName9 < indexedName10);
+
+    // Arrange
+    auto indexedName11 = Data::IndexedName("EDGE2");
+    auto indexedName12 = Data::IndexedName("EDGE12");
+
+    // Act & Assert
+    EXPECT_EQ(indexedName11.compare(indexedName12), -1);
```

Comment:
The exact number seems like an implementation detail (unless specifically stated in the docs). `...compare(...) < 0` seems more general.

Reply:
Good point: operator<=> is not guaranteed to return ±1. I will adjust these tests.